optimum/habana/accelerate/accelerator.py

```diff
@@ -59,6 +59,8 @@
 from accelerate.utils.other import is_compiled_module
 from torch.optim.lr_scheduler import LRScheduler
 
+from optimum.habana import parallel_state
```

Comment:
```suggestion
from .. import parallel_state
```
Let's use relative imports to stay consistent with the rest of the library

Reply:
Sure